src/storage.rs

```diff
@@ -72,4 +83,87 @@ impl BeerTally for HashMapBeerTally {
             },
         }
     }
+
+    fn get_players_vect(&mut self, chat_id: i64) -> Result<Vec<tally::UserTally>, &str> { 
+        if let Some(chat) = self.players.get_mut(&chat_id) {
```

Comment:
Changing from `get_mut` to `get` would allow you to not require a mutable reference for this method.

```suggestion
        if let Some(chat) = self.players.get(&chat_id) {
```